src/main/config/secor.dev.hr.partition.properties

```diff
@@ -0,0 +1,11 @@
+include=secor.dev.properties
+
+secor.kafka.group=secor_hr_partition
+secor.message.parser.class=com.pinterest.secor.parser.ThriftMessageParser
+
+secor.s3.path=secor_dev/hr_partition
+secor.local.path=/tmp/secor_dev/message_logs/hr_partition
+
+message.timestamp.using.hour=true
```

Comment:
In the spirit of separating parameter namespaces, could you rename it to
partitioner.granularity.hour
or something like that?